docs/src/dev/io/graphson.asciidoc

```diff
@@ -3580,6 +3572,41 @@ The following `Bytecode` example represents the traversal of `g.V().hasLabel('pe
 }
 ----
 
+==== P within
```

Comment:
Good idea to add `within` and `without` examples. Can we also add a little introductory text saying that `P` expects a single value or a list of values, except in the case of `within` and `without` where its always a list of values?

Reply:
that's a good idea. i will make a note to do it after i merge if that's ok. i'd rather not have to re-work the PR on master as a result of that little change. i have a bunch of cleanup to do anyway after these merge.